base/src/event_loop.cpp

```diff
@@ -1 +1,60 @@
 // Copyright Antioch. All rights reserved.
+
+#include "antioch/base/event_loop.h"
+
+#include <iostream>
+#include <thread>
+
+#include "antioch/base/config/configerator.h"
+#include "antioch/base/constants.h"
+
+namespace antioch::base {
+
+static int e = 1;
+
+EventLoop::EventLoop() : boot_time(std::chrono::system_clock::now()) {
+  tick = boot_time;
+  try {
```

Comment:
personally I do not like exceptions especially catch all exceptions since you start catching wonky stuff like bad_alloc and other stdlib exceptions that you might want to fail loudly instead

Reply:
in general I get not catching std::exception. but in this case I don't think we should fail loudly here. we can just use the default config and everything will be fine. maybe we should push it to the display but I think that's it